src/main/java/com/cloudbees/jenkins/support/AsyncResultCache.java

```diff
@@ -26,14 +26,14 @@
     private final Node node;
     private final String name;
 
-    public static <V, T extends java.lang.Throwable> V get(Node node, WeakHashMap<Node, V> cache, Callable<V,T> operation, String name, V defaultIfNull)
+    public static <V, T extends java.lang.Throwable> V get(Node node, WeakHashMap<Node, V> cache, /*MasterToSlave*/Callable<V,T> operation, String name, V defaultIfNull)
```

Comment:
Uh oh, maybe introducing a new API is reasonable in this case

Reply:
Seems like overkill I think—there are no known external usages.